cli/command/service.go

```diff
@@ -10,6 +18,260 @@ func NewServiceCommand(b *CommandBase) *ServiceCommand {
 	return &ServiceCommand{b}
 }
 
-func (e *ServiceCommand) Command() cli.Command {
-	return cli.Command{}
+func (s *ServiceCommand) Command() cli.Command {
+	return cli.Command{
+		Name:  "service",
+		Usage: "manage Layer0 services",
+		Subcommands: []cli.Command{
+			{
+				Name:      "create",
+				Usage:     "create a new service",
+				Action:    s.create,
+				ArgsUsage: "ENVIRONMENT NAME DEPLOY",
```

Comment:
[nit] We were having a discussion about the ArgsUsage `NAME` here becoming `SERVICE_NAME`. I think this is the currently agreed consensus on the discussions I saw about this.

Reply:
Oh cool, I support this. ✅